fastcrypto/src/bls12381.rs

```diff
@@ -254,21 +253,16 @@ impl VerifyingKey for BLS12381PublicKey {
             ));
         }
         let mut rands: Vec<blst_scalar> = Vec::with_capacity(sigs.len());
-        let mut rng = OsRng;
-
-        for _ in 0..sigs.len() {
-            let mut vals = [0u64; 4];
-            // TODO we should set at least 128 random bits, not just 64.
-            vals[0] = rng.next_u64();
-            while vals[0] == 0 {
-                // Reject zero as it is used for multiplication.
-                vals[0] = rng.next_u64();
-            }
-            let mut rand_i = MaybeUninit::<blst_scalar>::uninit();
-            unsafe {
-                blst_scalar_from_uint64(rand_i.as_mut_ptr(), vals.as_ptr());
-                rands.push(rand_i.assume_init());
-            }
+
+        let mut one = blst_scalar::default();
```

Comment:
Let's add a comment that this is an optimization, in accordance to https://github.com/MystenLabs/fastcrypto/issues/120

Reply:
It was put in this PR by accident, so I've removed it again. Will include it in another PR.

Reply:
@benr-ml just asked me why it is necessary with 128 bits here. The numbers are chosen by the verifier, so statistical security should be enough, right?